Makefile

```diff
@@ -23,6 +23,9 @@ install: libapriltag.so
 	@./install.sh $(PREFIX)/include/apriltag $(APRILTAG_HEADERS)
 	@ldconfig
 
+apriltag_config.h: apriltag_config.h.in
```

Comment:
@christian-rauch this generates a default apriltag_config.h (all #cmakefine lines will be define 1 lines) the default behavior for release builds of assert() will be unchanged. 

I don't think supporting a Makefile long term is sustainable. We should :hocho: it soon :)